### DISCUSSION TITLE
Phase one testing complete

### DISCUSSION THREAD

**Discussion by ferdiworks** [rrirower/DP2SceneGraphWizard#3] [General]
Michael

Thank you for the oppotunity. I have tested many DP feeds on my end with TV special and short form video and found the template put out with software to be working for me!

Found working with software is simplistic for non coders!

Looking forward to updates!

Next donations or funding support? Do you take any?

**Comment by rrirower** [comment]
Donations are not required.  But, if you'd like to support this project, you can donate from the wizard tool.

The first page of the wizard tool includes a "?" button on the bottom right. Click it. The next form contains a PayPal donate button on the bottom.

Alpha2 release should be available this weekend.

Thanks for your support.

**Reply by ferdiworks** [reply]
I guess I tested your paypal button on software and tried to get it to work and could not! So I manually took your paypal email and sent something, I've sent $100.00 USD to rrirower@charter.net

**Reply by rrirower** [reply]
Received. Thank you for your support!

Can you tell me what happened when you clicked the PayPal button?

**Reply by ferdiworks** [reply]
https://www.loom.com/share/5e0fe4c2860249d6aa8ac7f645da1484?sid=a8f33980-2f68-46ab-8771-33ed862ad77a

**Reply by rrirower** [reply]
Thanks for following up on this. That's a PayPal form and I believe it wants the user to enter an amount before it will process a click on the buttons.  The amount cannot be zero.

**Reply by ferdiworks** [reply]
Yes you are correct
https://www.loom.com/share/952ec8681a1d4b39aeba5a98cd312fda?sid=ff7a1303-9573-43a4-a0b4-4aa9727628f4
